Equilobe.DailyReport.SL/DataService.cs

```diff
@@ -482,7 +482,70 @@ public void SaveIpnLogError(long id, string error)
             }
```

Comment:
This class is actually an example of what I meant when I suggested you do the db calls in data service classes. This became huge so better in the future we break this class to more specific classes (like the ones we created recently).